src/common/redis_module/ray_redis_module.cc

```diff
@@ -656,6 +656,7 @@ int TableAdd_RedisCommand(RedisModuleCtx *ctx,
 int ChainTableAdd_RedisCommand(RedisModuleCtx *ctx,
                                RedisModuleString **argv,
                                int argc) {
+  RedisModule_AutoMemory(ctx);
```

Comment:
Did this fix the problem you were seeing with memory not being reclaimed even when flushing?

Reply:
That's right.